server/conf/i18n/messages

```diff
@@ -11,7 +11,21 @@
 button.logout=Logout
 label.selectLanguage=Please select your preferred language.
 button.untranslatedSubmit=Submit
+# Text on a link that gives more information about CiviForm. "CiviForm" is the app name and should
+# not be translated.
+footer.aboutCiviform=About CiviForm
 footer.supportLinkDescription=For technical support, contact
+# Text in the footer. The placeholder is a link call to action to visit a website.
+footer.lookingForMoreInformation=Looking for more information and services? {0}
```

Comment:
This is never used anywhere. Is that intentional?

Reply:
good catch it was supposed to be but we don't actually have the info needed for the placeholder value, so we're holding off for now. Filed https://github.com/civiform/civiform/issues/7959 for that chunk of work